src/CfdTransaction.cs

```diff
@@ -0,0 +1,157 @@
+using System;
+using System.Runtime.InteropServices;
+
+namespace Cfd
+{
+  //! txin sequence locktime
+  public enum CfdSequenceLockTime : uint
+  {
+    /// disable locktime
+    Disable = 0xffffffff,
+    /// enable locktime (maximum time)
+    EnableMax = 0xfffffffe,
+  };
+
+  /// <summary>
+  /// txid data class.
+  /// </summary>
+  public class Txid
+  {
+    private string txid;
+
+    public Txid(string txid)
+    {
+      this.txid = txid;
+    }
+
+    public Txid(byte[] txid)
+    {
+      byte[] temp_txid = CUtil.ReverseBytes(txid);
+      this.txid = StringUtil.FromBytes(temp_txid);
+    }
+
+    public string ToHexString()
+    {
+      return txid;
+    }
+
+    public byte[] ToBytes()
+    {
+      byte[] temp_txid = StringUtil.ToBytes(txid);
+      return CUtil.ReverseBytes(temp_txid);
+    }
+
+    public override int GetHashCode()
+    {
+      return txid.GetHashCode();
+    }
+
+    public override bool Equals(object obj)
+    {
+      var item = obj as Txid;
+      if (item == null)
+      {
+        return false;
+      }
+      return (txid == item.txid);
+    }
+
+    public static bool operator ==(Txid src, Txid dst)
```

Comment:
Operator overloading is not usually recommended in C#. Also I this implementation gives some weird behavior:
```
Txid txid = null;
Console.WriteLine(txid == null);
```
Output: False

Reply:
Thanks!